lbrynet/wallet/account.py

```diff
@@ -33,6 +33,7 @@ def get_certificate_private_key(self, ref: TXORef):
 
     @defer.inlineCallbacks
     def maybe_migrate_certificates(self):
+        yield self.save_max_gap()
```

Comment:
Can this be moved up one level and put next to where `maybe_migrate_certificate` is called?